mitosheet/src/mito/components/toolbar/Toolbar.tsx

```diff
@@ -71,103 +60,18 @@ export const Toolbar = (
-                        <ToolbarDataframesDropdown
-                            actions={props.actions}
-                            uiState={props.uiState}
-                            setUIState={props.setUIState}
-                            userProfile={props.userProfile}
-                        />
-                    </ToolbarMenu>
-                    <ToolbarMenu type='Columns' uiState={props.uiState} setUIState={props.setUIState}>
-                        <ToolbarColumnsDropdown
-                            actions={props.actions}
-                            uiState={props.uiState}
-                            setUIState={props.setUIState}
-                            userProfile={props.userProfile}
-                        />
-                    </ToolbarMenu>
-                    <ToolbarMenu type='Rows' uiState={props.uiState} setUIState={props.setUIState}>
-                        <ToolbarRowsDropdown
-                            actions={props.actions}
-                            uiState={props.uiState}
-                            setUIState={props.setUIState}
-                            userProfile={props.userProfile}
-                        />
-                    </ToolbarMenu>
-                    <ToolbarMenu type='Graphs' uiState={props.uiState} setUIState={props.setUIState}>
-                        <ToolbarGraphsDropdown
-                            actions={props.actions}
-                            uiState={props.uiState}
-                            setUIState={props.setUIState}
-                            userProfile={props.userProfile}
-                        />
-                    </ToolbarMenu>
-                    <ToolbarMenu type='Format' uiState={props.uiState} setUIState={props.setUIState}>
-                        <ToolbarFormatDropdown
-                            actions={props.actions}
-                            uiState={props.uiState}
-                            setUIState={props.setUIState}
-                            userProfile={props.userProfile}
-                        />
-                    </ToolbarMenu>
-                    <ToolbarMenu type='Code' uiState={props.uiState} setUIState={props.setUIState}>
-                        <ToolbarCodeDropdown
-                            actions={props.actions}
-                            uiState={props.uiState}
-                            setUIState={props.setUIState}
-                            userProfile={props.userProfile}
-                        />
-                    </ToolbarMenu>
-                    <ToolbarMenu type='View' uiState={props.uiState} setUIState={props.setUIState}>
-                        <ToolbarViewDropdown
-                            actions={props.actions}
-                            uiState={props.uiState}
-                            setUIState={props.setUIState}
-                            userProfile={props.userProfile}
-                        />
-                    </ToolbarMenu>
```

Comment:
I think you can remove all these ToolbarMenus! There is a lot of them, and some infrastructure for managing them you can chop (e.g. open dropdown).